pkg/errors/server.go

```diff
@@ -28,6 +28,14 @@ import (
 	"go.uber.org/yarpc/yarpcerrors"
 )
 
+var msgInboundDispatcherNotRunning = "peer for service %q is not running"
```

Comment:
"peer" has its own meaning within YARPC with its own class of objects. This should probably be "dispatcher" too?

Reply:
Changed, but keeping as separate functions for now. "Peer" was @kriskowal's idea, you two can hash it out heh